twill-api/src/main/java/org/apache/twill/internal/DefaultTwillRunResources.java

```diff
@@ -129,7 +153,7 @@ public String toString() {
       ", memoryMB=" + memoryMB +
       ", host='" + host + '\'' +
       ", debugPort=" + debugPort +
-      ", logLevel=" + logLevel +
+      ", rootLogLevel=" + logLevels.get(Logger.ROOT_LOGGER_NAME) +
```

Comment:
Why only has root log level? The `toString` should includes the whole `logLevels` map.